targets/PythonSdk/source/playfab/PlayFabSettings.py.ejs

```diff
@@ -33,6 +33,11 @@ This is automatically populated by the PlayFabAuthenticationApi.GetEntityToken m
 """
 _internalSettings.EntityToken = None
 
+"""
```

Comment:
As mentioned above, I don't think a new setting makes sense. We should just be storing the game server entity token in the existing EntityToken setting.